cypress/support/commands.js

```diff
@@ -23,3 +23,32 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.round(Math.random() * 1000);
+  const username = `user${randomNumber}`;
+  const lastname = `user${randomNumber}${randomNumber}`;
+  const email = `${username}@mail.com`;
+  const phone = Math.round(Math.random() * 10000000000);
```

Comment:
This line of code generates a random number for the phone, but it may not be a valid phone number. Consider using a library like faker to generate a valid phone number.